mkt/extensions/tests/test_views.py

```diff
@@ -513,22 +525,47 @@ def test_download(self):
 
     @override_settings(
         DEFAULT_FILE_STORAGE='mkt.site.storage_utils.S3BotoPrivateStorage')
-    def test_download_storage(self):
+    def test_download_signed_storage(self):
         ok_(self.extension.download_url)
         response = self.client.get(self.extension.download_url)
         path = public_storage.url(self.extension.signed_file_path)
         self.assert3xx(response, path)
 
-    def test_download_not_public(self):
+    def test_download_signed_not_public(self):
         self.extension.update(status=STATUS_PENDING)
         ok_(self.extension.download_url)
         response = self.client.get(self.extension.download_url)
         eq_(response.status_code, 404)
 
-        self.login(self.user)  # Even logged in you can't access it for now.
+        self.login(self.user)
+        self.grant_permission(self.user, 'Extensions:Review')
         response = self.client.get(self.extension.download_url)
+        # Even authors and reviewers can't access it: it doesn't exist.
         eq_(response.status_code, 404)
 
+    @override_settings(
+        XSENDFILE=True,
```

Comment:
No, that's why I'm forcing the storage. On the other views I've added a second test with the right storage class to test the S3 variant, but here I haven't. I'll do that.